### PR TITLE
WAVE2 1177 Enable new menu ui

### PR DESCRIPTION
**Pull request by robbdavis**
WAVE2 1177 Enable new menu ui

### CODE REVIEW

File: sites/all/themes/ucgy_theme/ckeditor.styles.js [hunk 1]
```diff
@@ -1,43 +0,0 @@
-/* The 'Styles' button is not enabled by default in DrupalFull and DrupalFiltered toolbars. */
```
Comment (gsowrey) [inline sites/all/themes/ucgy_theme/ckeditor.styles.js]: Honest question: Don't we need this for CKEditor?

Comment (TDRoden) [inline sites/all/themes/ucgy_theme/ckeditor.styles.js]: No, you're right - have restored 3 CKEditor files back into the theme folder that I missed - sorry about that.

File: sites/all/themes/ucgy_theme/resources/scss/navigation_hamburger.scss [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+      &::after {
+        bottom: 0;
+        transform: rotate(-45deg);
+      }
+    }
+  }
+}
+
+//
```
Comment (dpfitzsi) [inline sites/all/themes/ucgy_theme/resources/scss/navigation_hamburger.scss]: Should this code (below) be deleted?

File: sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js [hunk 3]
```diff
@@ -70,62 +70,60 @@ var ucgy_analytics;
...
+            $('#navigation li.children a').on('click', function () {
                 var link_title = $(this).text();
-                var attr = $(this).attr('aria-expanded');
-                if(typeof attr !== 'undefined' && attr === 'true') {
-                    ga(handle, 'event', 'Menu', 'Close', link_title);
-                }else{
+                if (link_title == 'Back') {
+                    ga(handle, 'event', 'Menu', 'Back', link_title);
+                }
+                else if (!$(this).parent().hasClass('active')) {
```
Comment (dpfitzsi) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: should be elseif

File: sites/all/themes/ucgy_theme/resources/scss/navigation_hamburger.scss (hunk 2 above)
Comment (TDRoden) [inline sites/all/themes/ucgy_theme/resources/scss/navigation_hamburger.scss]: These are specific hamburger animation styles that I would like to keep record of - they are commented out and therefore not compiled so don't see any harm.

File: sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js (hunk 3 above)
Comment (TDRoden) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: Not in JS.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.install [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * Disable and uninstall the crumbs module.
+ */
+function ucgy_core_update_7001() {
+  if( module_exists('crumbs')) {
```
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.install]: Have adjusted Robb's code as per our usual formatting.

File: sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js [hunk 5]
```diff
@@ -257,19 +268,19 @@ var ucgy_analytics;
...
             $('#program-search')
-                .find('input#programs').on('focus', function (evt) {
+                .find('input#programs').on('focus', function () {
                 ga(handle, 'event', 'Program search', 'Focus', '');
-
             }).end()
                 .find('button[type="submit"]').on('click', function () {
                 ga(handle, 'event', 'Program search', 'Trigger', document.getElementById('programs').value);
             });
-            //added to stop extra focus events from firing
```
Comment (taeram) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: @TDRoden This change reverts the core fix in WAVE2-1201 (https://github.com/ucalgary/www.ucalgary/pull/785/files#diff-036c2ddf842de0e731e2406f76a653c9R258).

If this was accidental, please re-add lines 269-272.

Comment (taeram) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: @TDRoden Aaaand I just realized you've re-added this code a few lines up. No changes needed!

**Pull request merged**